azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -601,10 +601,9 @@ private void doRun() {
       finalStatus = changeStatus(Status.KILLED);
     }
 
-    logInfo(
-        "Finishing job " + this.jobId + getNodeRetryLog() + " at " + this.node.getEndTime()
-            + " with status " + this.node.getStatus());
-
+    final int attemptNo = this.node.getAttempt();
```

Comment:
Why this change? @chengren311 refactored this code earlier.

Reply:
Accidental, I guess while I rebased. Will remove this.

Reply:
done.